parsers/html.c

```diff
@@ -157,29 +158,31 @@ typedef enum {
 	TOKEN_NAME,			/* tag and attribute names */
 	TOKEN_STRING,		/* single- or double-quoted attribute value */
 	TOKEN_TEXT,
-	TOKEN_TAG_START,	/* <  */
-	TOKEN_TAG_START2,	/* </ */
+	TOKEN_OPEN_TAG_START,	/* <  */
+	TOKEN_CLOSE_TAG_START,	/* </ */
 	TOKEN_TAG_END,		/* >  */
 	TOKEN_TAG_END2,		/* /> */
 	TOKEN_EQUAL,
 	TOKEN_COMMENT,
-	TOKEN_OTHER
+	TOKEN_OTHER,
+	TOKEN_OTHER_SCRIPT,
```

Comment:
TOKEN_OTHER_SCRIPT is introduced here. However, it seems that this one is not used in this pull request.

Reply:
Removed.  It was left over from a previous iteration of the code.